lib/rack/cache/entitystore.rb

```diff
@@ -336,6 +336,38 @@ def self.resolve(uri)
     GAECACHE = GAEStore
     GAE = GAEStore
 
+    # Dummy Entity Store that does not persist entities
+    class Dummy < EntityStore
+      def exist?(key)
+        true
+      end
+
+      def read(key)
+        nil
+      end
+
+      def open(key)
+        nil
```

Comment:
Not without changing ```MetaStore#store```. 

For some reason I don't understand, line 69 of that method does this:

```
response.body = entity_store.open(digest) || response.body
```

This is, just after writing the response in the cache, the body is read from the cache and the response body is overwritten with whatever is read from the entity store, unless ```entity_store.open(digest)``` returns nil. 

If ```Dummy#open``` returns ```[ ]``` (or anything other than nil, really), the response body will be overwritten with ```[ ]``` even for requests that miss the cache. This means that the actual response body will **never** be returned by the library, not even for the first request (which results in a cache miss and the response being stored in the cache).

The only solutions are either to make ```Dummy.open``` return nil so that the response body is not overwritten, or to get rid of line 69. If we remove line 69 of ```MetaStore```, ```Dummy``` can return ```[ ]``` instead of nil.

I can't see the reason for that line, I don't think it makes sense to read the response body from the cache immediately after saving it in the cache. But I may be missing something important there. What do you think?